processes.md

```diff
@@ -0,0 +1,202 @@
+
+See our [contributor ladder](#contributor-ladder) for more information.
+
+## Contributor Ladder
+
+<!--Big thanks to the folks at https://github.com/cncf/project-template
+for providing the base framework for this section.-->
+
+This contributor ladder outlines the different contributor roles within the project, along with the responsibilities and privileges that come with them. Community members generally start at the first levels of the "ladder" and advance up it as their involvement in the project grows. Our project members are happy to help you advance along the contributor ladder.
+
+Each of the contributor roles below is organized into lists of three types of things. "Responsibilities" are things that contributor is expected to do. "Requirements" are qualifications a person needs to meet to be in that role, and "Privileges" are things contributors on that level are entitled to.
+
+### Community Participant
+
+Description: A Community Participant engages with the project and its community, contributing their time, thoughts, etc. Community participants are usually users who have stopped being anonymous and started being active in project discussions.
+
+* Responsibilities:
+  * Must follow the [CDEvents Code of Conduct](https://github.com/cdevents/.github/blob/main/docs/CODE_OF_CONDUCT.md)
+* How users can get involved with the community:
+  * Participating in community discussions
+    ([GitHub, Slack, mailing list, etc](/README.md#how-to-get-involved))
+  * Helping other users
+  * Submitting bug reports
+  * Commenting on issues
+  * Trying out new releases
+  * Attending community events
+
+### Contributor
+
+Description: A Contributor makes direct contributions to the project and adds value to it. [Contributions need not be code](#contributions). People at the Contributor level may be new contributors, and they can contribute occasionally.
+
+Contributors may be eligible to vote and run in elections. See [Elections](/governance.md#elections) for more details.
+
+A Contributor must meet the responsibilities of a [Community Participant](#community-participant), plus:
+
+* Responsibilities include:
+  * Follow the project contributing guide (see `CONTRIBUTING.md` in the
+    corresponding repo)
+* Requirements (one or several of the below):
+  * Report and sometimes resolve issues
+  * Occasionally submit PRs
+  * Contribute to the documentation
+  * Participate in [meetings](/README.md#how-to-get-involved)
+  * Answer questions from other community members
+  * Submit feedback on issues and PRs
+  * Test, review, and verify releases and patches
+* Privileges:
+  * Invitations to contributor events
+  * Eligible to become an [Organization Member](#organization-member)
+
```

Comment:
What is the process to become a Contributor? Where are contributors kept track of?

Reply:
There is no "process" for becoming a contributor. Whoever contributes to CDEvents in various forms is a contributor. 

There is no easy way to keep track of contributors either, unfortunately. 
One way we achieve that is through devstats, but that does not necessarily cover everything. 
Asking people to register or fill out a form before making a contribution would raise the bar to contributions, and that's not something we want.

In case of elections though, we may ask people to fill a form to declare themselves as contributors if they want to vote and are not tracked by devstat

Reply:
That's all good. I guess I was considering e.g. the privilege of being invited to contributor events (whatever that is), since then we would need a well-defined list of people with contact info to be able to reach them. But I guess we could copy that from devstats. I'm glad we don't add extra admin to manage the list of contributors manually.